content/60.methods.md

```diff
@@ -2,6 +2,79 @@
 
 ### Article Selection and Evaluation
 
+The authors solicited relevant articles to be submitted via GitHub [@url:https://Github.com] for review. 
```

Comment:
```suggestion
The authors solicited relevant articles to be submitted via [GitHub](https://github.com/greenelab/covid19-review) for review. 
```

What do you think about linking to this repo instead of the main GitHub site?  I also tend to use standard Markdown links instead of Manubot citations for URLs like this that don't have specific authors or dates associated with them.